### PR TITLE
Feature/display adhooc or scheduled

### PR DESCRIPTION
**Pull request by Angatupyry**
## What's new

- Display on task queue whether task was scheduled or ad-hoc
- For now, automatic robot tasks do not have a distinct icon due we have not a way to differential it from real users since the name is the ROS 2 node name

chrome-capture-2023-6-31

## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: packages/react-components/lib/tasks/task-table-datagrid.tsx [hunk 1]
```diff
@@ -134,11 +135,24 @@ export function TaskDataGridTable({
...
+        return (
+          <Stack direction="row" alignItems="center" gap={1}>
+            {cellValues.row.booking.requester &&
+            cellValues.row.booking.requester.includes('scheduled') ? (
+              <ScheduleIcon />
+            ) : (
+              <UserIcon />
+            )}
+            <Typography variant="body1">
+              {cellValues.row.booking.requester ? cellValues.row.booking.requester : 'unknown'}
```
Comment (aaronchongth) [inline packages/react-components/lib/tasks/task-table-datagrid.tsx:150]: We can probably just display `stub` instead of `stub__scheduled` here, the different icon should be a good enough indicator

File: packages/react-components/lib/tasks/task-table-datagrid.tsx [hunk 2]
```diff
@@ -134,11 +135,24 @@ export function TaskDataGridTable({
...
       editable: false,
-      valueGetter: (params: GridValueGetterParams) =>
-        params.row.booking.requester ? params.row.booking.requester : 'unknown',
-      flex: 1,
+      renderCell: (cellValues) => {
+        return (
+          <Stack direction="row" alignItems="center" gap={1}>
+            {cellValues.row.booking.requester &&
+            cellValues.row.booking.requester.includes('scheduled') ? (
+              <ScheduleIcon />
```
Comment (aaronchongth) [inline packages/react-components/lib/tasks/task-table-datagrid.tsx:145]: Can we add a tooltip to these icons, such that hovering over them tells us what the icons mean. We can have the tooltips display `User scheduled`, as well as `User submitted` for the case of `UserIcon`.

**Review by aaronchongth — CHANGES_REQUESTED** [review]
These look great! Thanks for adjusting the width of the table too. I left some minor comments on how it looks, but the implementation works great!

File: packages/react-components/lib/tasks/task-table-datagrid.tsx (hunk 1 above)
Comment (Angatupyry) [reply packages/react-components/lib/tasks/task-table-datagrid.tsx]: Sure!
Done 17b513d8de25bf91f8fe4136401a830095bdf5fe

**Review by aaronchongth — APPROVED** [review]
LGTM! Thanks!
